.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
```

Comment:
I prefer following configuration:
```suggestion
# Keep lineendings except for Git internal files.
.git* text=auto
* -text

```

Reply:
Then the shell scripts wouldn't correctly be checked out, where git is configured to use windows line endings for text files (which shell scripts are). I don't mind adding the other lines, but mine is still required to make the tests work regardless of the environment.

Reply:
'-text' removes this setting and keep s the line endings as commited.

Reply:
So since the line endings are correctly committed, it would leave the shell scripts with Unix style endings?